test/schema.test.ts

```diff
@@ -38,9 +38,10 @@ describe('Notices file is valid', () => {
       });
 
       test('v2 version ranges must be bounded at the bottom', () => {
+        const v2Range = '>=2.0.0-rc.1';
```

Comment:
@rix0rrr Can you confirm this looks ok?

Reply:
It doesn't, because this will also match `3.0.0`. I think. At this point I'm questioning everything I know 😅

Reply:
```suggestion
        const v2Range = '>=2.0.0-rc.1 <3.0.0';
```

?